### DISCUSSION TITLE
Affiliates?

### DISCUSSION THREAD

**Discussion by jtoelle** [Glench/ExtPay#135] [General]
Has anyone figured out a way to offer Affiliates commissions for promoting your extension with Extensionpay?

**Comment by rurimiao** [comment]
have the same problem, it is important to marketing!

**Reply by rurimiao** [reply]
and how can I increase subscription duration for users? for affiliates or gift to users.

**Reply by headturnermedia** [reply]
I'm going to try Rewardful.  Not sure if it works yet.

**Reply by EDDYYUNGMEDIA** [reply]
Please  what's rewardful

**Comment by benf2004** [comment]
Reflio looks interesting.

**Comment by benjang032** [comment]
Affiliates or even just discount codes would be very helpful

**Comment by BowTiedSwan** [comment]
I'd love to have this because I have a community of people that could promote my extension.
The issue of Rewardful (probably) is that creating a product and connecting it to their system would bypass the mechanism Extensionp pay uses to get the 5% commission, and this product won't be the same as what Extensionpay programmatically creates from the user dashboard.

If someone can find an open source project for doing affiliates it might be possible to make it work with extensionpay

**Comment by Glench** [comment]
Discount codes are now possible with ExtPay 3.1.0 (and 3.1.1)!

**Reply by burkybang** [reply]
I don't think this topic should be closed yet. Discount codes and affiliates have some overlap, but aren't quite the same thing.

**Reply by BowTiedSwan** [reply]
Agree.
Affiliates is a completely different functionality in that it has two main features:
- Allows tracking and assignment of payment subscriptions to a specific product link, which is linked to an affiliate ID that has at least an email associated with it (the affiliate wanting to promote our extension)
- Automates payments of affiliate commissions every week/month or X frequency
